scripts/check-release-please.mjs

```diff
@@ -0,0 +1,82 @@
+import { strict as assert } from 'assert';
+import * as childProcess from 'child_process';
+import path from 'path';
+import { readFileSync } from 'fs';
+
+/*
+	Enumerates through
+
+	- lerna packages,
+	- release please's manifest, and
+	- release please's config, and
+
+	makes sure all those lists are in sync. If not, guides you to do the correct thing
+*/
+
+const errors = [];
+const logErrorIf = (condition, errorMessage) => {
+	if (condition) {
+		errors.push(errorMessage);
+	}
+};
+const PROJECT_ROOT = path.resolve();
+const readJson = (path) => {
+	return JSON.parse(readFileSync(path));
```

Comment:
hiding the imported `path` here is confusing

Reply:
Renamed the argument. :+1: